spec/core/SpyStrategySpec.js

```diff
@@ -92,16 +92,40 @@ describe("SpyStrategy", function() {
     expect(returnValue).toEqual(67);
   });
 
+  it("allows a fake async function to be called instead", async function(done) {
```

Comment:
`async` here is unnecessary because the spec function takes a callback, and it'll break on platforms that don't support it. In general, asynchronous specs should do any one of: take a callback, explicitly return a promise, or be declared `async`. But in Jasmine's own test suite we need to stick to the callback style.